labs-ios-starter/Surveys/View Controllers/TopicQuestionsViewController.swift

```diff
@@ -94,7 +94,7 @@ class TopicQuestionsViewController: UIViewController {
         }
 
         let selected = contextSegmentControl.selectedSegmentIndex
-
+        #warning("Error 500 here")
```

Comment:
This should be resolved

Reply:
Yeah I'm going to open it up on a new PR, I marked it here so I wouldn't forget, but I think it makes more sense to open an issue rather than commit a warning into the master branch. I'll do that.